### PR TITLE
Build ienn binary with ninja action

### PR DESCRIPTION
**Pull request by fujunwei**

### CODE REVIEW

File: src/webnn_native/openvino/ienn/build_ienn_msvc.bat [hunk 1]
```diff
@@ -134,25 +120,32 @@ if "!MSBUILD_BIN!" == "" (
    GOTO errorHandling
 )
 
-if exist "%SOLUTION_DIR64%\CMakeCache.txt" del "%SOLUTION_DIR64%\CMakeCache.txt"
+:: Create vs project will take long time that isn't necessary.
+::if exist "%SOLUTION_DIR64%\CMakeCache.txt" del "%SOLUTION_DIR64%\CMakeCache.txt"
```
Comment (huningxin) [inline src/webnn_native/openvino/ienn/build_ienn_msvc.bat:124]: Please remove the unused code.

File: src/webnn_native/openvino/ienn/build_ienn_unix.sh [hunk 2]
```diff
@@ -72,10 +79,13 @@ cd "$build_dir"
 cmake -DCMAKE_BUILD_TYPE=Release "$CURRENT_PATH"
 make $NUM_THREADS
 
-# copy the libie_nn_c_api.so to ienn\lib\Linux64
-if [ -e "$CURRENT_PATH/lib/Linux64/libie_nn_c_api.so" ]; then
-    rm -rf "$CURRENT_PATH/lib/Linux64/libie_nn_c_api.so"
+# copy the libie_nn_c_api.so to webnn_native\out\Release
```
Comment (huningxin) [inline src/webnn_native/openvino/ienn/build_ienn_unix.sh:82]: Is the "webnn_native\out\Release" just an exmaple? Should we make this comment more generic?

File: src/webnn_native/openvino/ienn/build_ienn_msvc.bat [hunk 3]
```diff
@@ -134,25 +120,32 @@ if "!MSBUILD_BIN!" == "" (
...
 echo "!MSBUILD_BIN!" ienn.sln /p:Configuration=Release
 "!MSBUILD_BIN!" ienn.sln /p:Configuration=Release
 if ERRORLEVEL 1 GOTO errorHandling
 
-:: copy the ie_nn_c_api.dll to ienn\lib\Windows64
-if exist "%ROOT_DIR%\lib\Windows64\ie_nn_c_api.dll" del "%ROOT_DIR%\lib\Windows64\ie_nn_c_api.dll"
-xcopy "%ROOT_DIR%\build\intel64\Release\ie_nn_c_api.dll" "%ROOT_DIR%\lib\Windows64"
-xcopy "%ROOT_DIR%\build\intel64\Release\ie_nn_c_api.lib" "%ROOT_DIR%\lib\Windows64"
-echo Done.
+:: copy the ie_nn_c_api.dll to webnn_native\out\Release
```
Comment (huningxin) [inline src/webnn_native/openvino/ienn/build_ienn_msvc.bat:138]: ditto, please make the comment more generic.

**Review by huningxin — COMMENTED** [review]
Thanks @fujunwei . Besides the build script, I am wondering whether is possible to move the ienn to third_party, as partially address #2.

**Comment by fujunwei** [comment]
> Thanks @fujunwei . Besides the build script, I am wondering whether is possible to move the ienn to third_party, as partially address #2.

Do, thanks, please take a look again.

**Review by huningxin — COMMENTED** [review]
Code looks good. Two comments about the project setup:
1. Should we move this component to `//third_party/openvino/ienn`? It would leave rooms for other projects in the future, e.g., openvino source code itself.
2. Please add a README.md that describes this component. 

Thanks!

**Comment by fujunwei** [comment]
> Code looks good. Two comments about the project setup:
> 
> 1. Should we move this component to `//third_party/openvino/ienn`? It would leave rooms for other projects in the future, e.g., openvino source code itself.
> 2. Please add a README.md that describes this component.
> 
> Thanks!

That's good suggestion. Please take a look the README, thanks.

**Comment by fujunwei** [comment]
> It's not to build nGraph API. And please list the steps to build this component, e.g.,
> 
> Please install OpenVINO 2021.2 version.
> 
> Setup OpenVINO environmental variables.
> 
> Execute `build_ienn_msvc.bat` on Windows or `build_ienn_unix.sh` on Linux.

Done, thanks, please take a look.

**Review by huningxin — APPROVED** [review]
Looks good, thanks @fujunwei !

**Pull request merged**